snow/consensus/snowball/factory.go

```diff
@@ -6,8 +6,8 @@ package snowball
 import "github.com/ava-labs/avalanchego/ids"
 
 var (
-	SnowballFactory  Factory = snowballFactory{}
-	SnowflakeFactory Factory = snowflakeFactory{}
+	FactoryInstance Factory = snowballFactory{}
+	_               Factory = snowflakeFactory{}
```

Comment:
@aaronbuchwald Do we use `SnowflakeFactory` in some other repo?

I think we added this to test out the `Tree` using just `Snowflake` rather than using `Snowball`... Does this change break anything?

Reply:
> Do we use `SnowflakeFactory` in some other repo?

afaict according to [this search](https://github.com/search?q=org%3Aava-labs+SnowflakeFactory&type=code) across ava-labs repos, it is only used in avalanchego

Reply:
iirc I was using this in the simulator repo, but am not continuing that work, so it's fine to remove imo